test/functionalTest/cases/1156_qfilters_and_compounds/qfilters_and_compounds_deeper.test

```diff
@@ -653,7 +653,7 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36})
                             "F": false, 
                             "N": null, 
                             "T": true, 
-                            "f": 3.1400000000000001, 
+                            "f": 3.14,
```

Comment:
Probably, but I cannot check that, as the problem it seems that only happens in your system...

I propose the following: after merging this PR, try to change `3.14REGEX(...)` to just `3.14` and check if it works in your system.